tests/x11/remote_desktop/windows_client_remotelogin.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Remote Login: Windows access SLES over RDP
```

Comment:
I don't think it needs to be limited to SLES, not even SLE. How about "GNU/Linux" or "openSUSE/SLE"?

Reply:
Yes, "openSUSE/SLE" is a better choice in the long run. Will update the summary. Thanks.

Reply:
Updated.